sbt-conductr-tester/lagom-bundle/build.sbt

```diff
@@ -9,7 +10,10 @@ lazy val `lagom-service-impl` = (project in file("lagom-service-impl"))
   .dependsOn(`lagom-service-api`)
 
 lazy val `play-service` = (project in file("play-service"))
-    .enablePlugins(PlayJava, LagomPlay)
-    .settings(
-      routesGenerator := InjectedRoutesGenerator
-    )
\ No newline at end of file
+  .enablePlugins(PlayJava && LagomPlay)
+  .settings(
+    routesGenerator := InjectedRoutesGenerator,
+    // Need to explicitly add the dependency to lagom-client 1.3.1 because 1.3.1-RC1 is being pulled in
```

Comment:
Unfortunately I still had dependency override and had to explicitly import lagom-client with the correct version.